src/natcap/invest/carbon.py

```diff
@@ -91,48 +91,51 @@
             "type": "csv",
             "required": True,
             "about": (
-                "A table that maps the land-cover IDs to carbon pools.  "
-                "The table must contain columns of 'LULC', 'C_above', "
-                "'C_Below', 'C_Soil', 'C_Dead' as described in the User's "
-                "Guide.  The values in LULC must at least include the LULC "
-                "IDs in the land cover maps."),
+                "A table that maps each land cover class to its respective carbon pools.  "
+
+                "The table must contain columns of 'lucode', 'C_above', "
+
+                "'C_below', 'C_soil', 'C_dead' as described in the User's "
+
+                "Guide. The 'lucode' column must include all the pixel values "
+
+                "present in the land cover maps."),

```

Comment:
@davemfish this might just be my github window, but I think there are some bonus blank lines here ... are those showing up for you as well?

Reply:
Oh weird, these did not show up in the diffs until now. Thanks.